styles.css

```diff
@@ -315,6 +331,118 @@ menu {
   background: #008552;
 }
 
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  display: none;
+  height: 100vh;
+  background-color: #c1c7d0;
+  z-index: 9999;
+}
+
+.modal_item {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  width: 100%;
+  height: 100%;
+  align-items: flex-start;
+}
+
+.modal_img_container {
+  position: relative;
+  width: 100%;
+}
```

Comment:
Kindly ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins). 
For the mobile version, kindly ensure that the modal covers the whole page, also the image and the close button should be just like in the figma design.

### Your mobile design

![Screenshot from 2022-12-21 07-10-24](https://user-images.githubusercontent.com/65233860/208845009-d81de907-b298-4350-a3f6-1aba9f5b1255.png)

### Your desktop design 

![Screenshot from 2022-12-21 07-10-52](https://user-images.githubusercontent.com/65233860/208845036-d0b0487f-e16e-45c7-a83f-47d457e93490.png)


### Figma mobile design


![Screenshot from 2022-12-21 07-23-39](https://user-images.githubusercontent.com/65233860/208844984-e07cb113-afd0-4f2a-8efb-6f311b1803df.png)


### Figma desktop design


![Screenshot from 2022-12-21 07-24-11](https://user-images.githubusercontent.com/65233860/208845843-d72e40e9-14bc-46e8-a01b-b84c1003adec.png)